### ISSUE TITLE
Migrate fails when using community edition due to isSkipExecutingMigrations() being called

### ISSUE THREAD

**Issue by wilkinsona** [flyway/flyway#2991]
###### Which version and edition of Flyway are you using?

7.1.0
###### If this is not the latest version, can you reproduce the issue with the latest one as well?
(Many bugs are fixed in newer releases and upgrading will often resolve the issue)

N/A

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)

Java API

###### Which database are you using (type & version)?

N/A

###### Which operating system are you using?

N/A

###### What did you do?

Called `migrate()` with some migrations to run.
###### What did you expect to see?

The migrations were applied to the database.

###### What did you see instead?

A [failure due to `isSkipExecutingMigrations()` being called](https://ge.spring.io/s/yqsry63ieezfa/tests/:spring-boot-project:spring-boot-autoconfigure:test/org.springframework.boot.autoconfigure.quartz.QuartzAutoConfigurationTests/withFlyway(Path)#1):

```
Caused by: org.flywaydb.core.internal.license.FlywayTeamsUpgradeRequiredException: Flyway Teams Edition upgrade required: skipExecutingMigrations is not supported by Flyway Community Edition.	
	 	at org.flywaydb.core.api.configuration.ClassicConfiguration.isSkipExecutingMigrations(ClassicConfiguration.java:704)	
	 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:184)	
	 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:164)	
	 	at org.flywaydb.core.Flyway.execute(Flyway.java:534)	
	 	at org.flywaydb.core.Flyway.migrate(Flyway.java:164)	
	 	at org.springframework.boot.autoconfigure.flyway.FlywayMigrationInitializer.afterPropertiesSet(FlywayMigrationInitializer.java:66)	
	 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1847)	
	 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1784)	
	 	... 127 more
```

I believe this is due to the changes made in https://github.com/flyway/flyway/commit/0e6fd6282ef345297802a15bb40502fb323c7a66. We discovered the problem when attempting to upgrade Spring Boot to Flyway 7.1.0.

**Comment by MikielAgutu** [comment]
Sorry about this! We will ship a fix ASAP. Thanks for bringing it to our attention.

**Comment by MikielAgutu** [comment]
This should be fixed in 7.1.1 https://flywaydb.org/documentation/learnmore/releaseNotes#7.1.1

**Comment by wilkinsona** [comment]
Thanks very much.
